.gitignore

```diff
@@ -186,6 +186,7 @@ $RECYCLE.BIN/
 /source/Makefile.in
 /unix/Makefile.am
 /unix/Makefile.in
+/unix/config/compile
```

Comment:
The cleanup of automake artifacts in `unix/prebuild.sh` should probably be updated accordingly (search for `config.guess`)